lib/nimble_template/addons/github.ex

```diff
@@ -36,16 +36,20 @@ defmodule NimbleTemplate.Addons.Github do
       web_project?: web_project?
     ]
 
-    template_file_path =
-      if mix_project? do
-        ".github/workflows/test.yml.mix.eex"
-      else
-        ".github/workflows/test.yml.eex"
-      end
-
-    files = [
-      {:eex, template_file_path, ".github/workflows/test.yml"}
-    ]
+    files =
+      [{:text, ".github/workflows/README.md", ".github/workflows/README.md"}] ++
+        if mix_project? do
+          [
+            {:eex, ".github/workflows/test.yml.mix.eex", ".github/workflows/test.yml"},
+            {:eex, ".github/workflows/deploy_heroku.yml.mix.eex",
```

Comment:
the normal mix project doens't work on Heroku I think @longnd, it requires at least the Plug lib in order to run on Heroku because heroku only support web-interface application while a normal Mix application isn't

Reply:
Fixed in b8d19bf.